test/WebJobs.Script.Tests/HttpWorker/HttpWorkerTestUtilities.cs

```diff
@@ -33,6 +33,8 @@ public static HttpRequest GetTestHttpRequest()
         {
             var httpRequest = new DefaultHttpContext().Request;
             httpRequest.Method = "GET";
+            httpRequest.Scheme = "http";
+            httpRequest.Host = new HostString("localhost", 8080);
```

Comment:
Is it possible to take input string for hoststring? that will allow us to add more tests on request path. If too many changes we can skip it.

Reply:
Yeah, added input string